app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+
+<div class='form-page'>
+  <h2>Sign up</h2>
+  <div class="form-container">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="field">
+        <%= f.label :name %><br />
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", class: 'input' %>
+      </div>
+
+      <div class="field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'input' %>
+      </div>
+
```

Comment:
- [ ] Same with registration page. Please follow the [given design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). The input field is different and that's okay but keep design same.

### Your design

![Screenshot from 2021-12-04 17-54-25](https://user-images.githubusercontent.com/22434660/144709275-2f219de9-b3b3-450f-a705-1392e022da24.png)



### Given design

![Screenshot from 2021-12-04 17-51-11](https://user-images.githubusercontent.com/22434660/144709261-eb7abad3-d37c-4420-a906-32e57d694ab2.png)